tests/BuildScriptGenerator.Tests/Node/NodePackageScriptsCheckerTest.cs

```diff
@@ -36,10 +36,12 @@ public void Checker_DetectsGlobalNpmInstalls()
         [InlineData("npm install --global pkg", true)]
```

Comment:
Add a case like `npm install foo -g` and `npm install foo --global` (basically package name before the global switch)